packages/lib/services/e2ee/EncryptionService.ts

```diff
@@ -99,14 +89,45 @@ export default class EncryptionService {
 		return Object.keys(this.decryptedMasterKeys_).length;
 	}
 
-	public chunkSize() {
-		return this.chunkSize_;
+	// Note: 1 MB is very slow with Node and probably even worse on mobile.
+	//
+	// On mobile the time it takes to decrypt increases exponentially for some reason, so it's important
+	// to have a relatively small size so as not to freeze the app. For example, on Android 7.1 simulator
+	// with 4.1 GB RAM, it takes this much to decrypt a block;
+	//
+	// 50KB => 1000 ms
+	// 25KB => 250ms
+	// 10KB => 200ms
+	// 5KB => 10ms
+	//
+	// So making the block 10 times smaller make it 100 times faster! So for now using 5KB. This can be
+	// changed easily since the chunk size is incorporated into the encrypted data.
```

Comment:
**🟣 Commenting**: Consider updating this comment &mdash; does it apply to the new encryption methods.

Reply:
I have added `Note for methods using SJCL:` for this piece of notes.

I also find that the optimum chunk size differs for StringV1 method and FileV1 method. I think I need some time to do a full evaluation.